CMakePresets.json

```
@@ -45,8 +45,28 @@
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "RelWithDebInfo"
       }
-    }
-  ],
+    },
+    {
+      "name": "linux-default",
+      "displayName": "Linux Debug",
+      "description": "Sets Ninja generator, compilers, build and install directory, debug build type",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/out/build/${presetName}",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "SELF_BUILT_AUBIO": "ALWAYS",
+        "SELF_BUILT_CED": "ALWAYS",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/out/install/${presetName}"
```

Please use the same structure as windows
```suggestion
       "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/${presetName}-install"
```

---

👍